voltixApp/VoltixApp/Views/Keysign/KeysignDiscoveryView.swift

```diff
@@ -196,12 +196,15 @@ struct KeysignDiscoveryView: View {
         return Image(systemName: "xmark")
     }
     
-    private func handleSelection(_ peer: String) {
+    func handleSelection(_ peer: String) {
         isLoading = true
         
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
```

Comment:
why we need asyncAfter?

Reply:
Not exactly sure about that , @Rockindash probably knows